### ISSUE TITLE
Feature Request: Provide set-up guidance to new users

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#6]
As a new user of Buoy, I found being immediately dropped into the Wordpress profile screen a bit confusing -- particularly because the Admin Color Scheme and Show Toolbar options at the very top are not typical options I am used to seeing when installing a new app. (Is there any way to remove these and start the profile at the Name section?)

The first time a new user logs into Buoy, consider displaying the following text either in a pop-up or at the top of the profile page:

"Welcome to Buoy. The Buoy tool connects people in crisis with the people they trust. In order to be available to respond to others in times of need, you must tell Buoy how you can be reached in an emergency. By default, all users receive crisis alerts by e-mail. To also receive crisis alerts by SMS, please supply your phone number in the field below. To create your personal emergency response team, click the My Team button in the left sidebar."

**Comment by fabacab** [comment]
I agree we should have a "set up wizard" or guide that interactively walks people through the Buoy setup process, or at least some parts of the [Buoy Walkthrough](https://github.com/meitar/better-angels/wiki/Buoy-Walkthrough).

For now, since the interface is still changing, I think we should do the same thing as we did for Predator Alert Tool for Facebook, which is simply embed help links to guides that we write on this wiki. Those guides should exist and they current don't, so if you feel up to it, I'd you to create wiki page stubs for all the guides you think we need and then create issues for me to add helpful links to them from the plugin's code itself.

I'll write the [Security advice](https://github.com/meitar/better-angels/wiki/Security-advice) guide (already linked to from [this FAQ](https://github.com/meitar/better-angels/wiki/Frequently-Asked-Questions#can-people-i-didnt-invite-see-my-notifications) but not yet actually written.

Once the UI stabilizes, I can code a more interactive walk through that will do things like highlight buttons on the screen in a sequence that explains what Buoy's features are and how to bookmark the alert screen to "install" the "app" on your "phone" and whatever else people are indoctrinated by capitalism to think they need to do with this tech for it to work.

**Comment by fabacab** [comment]
Note to self: use WP's built in "pointer content" features. Tutorial here: http://code.tutsplus.com/articles/integrating-with-wordpress-ui-admin-pointers--wp-26853

**Comment by hurtstotouchfire** [comment]
I'd like to tentatively start looking at what it would take to do this. I think we should also, ideally, find ways to simplify the user setup such that this is less necessary, but starting to look at building a wizard might help with that.
